src/main/java/org/jitsi/impl/protocol/xmpp/XmppProtocolProvider.java

```diff
@@ -25,10 +25,13 @@
 
 import org.jitsi.eventadmin.*;
 import org.jitsi.impl.protocol.xmpp.colibri.*;
+import org.jitsi.jicofo.FocusBundleActivator;
+import org.jitsi.jicofo.FocusManager;
 import org.jitsi.jicofo.recording.jibri.*;
 import org.jitsi.protocol.xmpp.*;
 import org.jitsi.protocol.xmpp.colibri.*;
 import org.jitsi.retry.*;
+import org.jitsi.service.configuration.ConfigurationService;
 import org.jitsi.util.Logger;
```

Comment:
@guusdk and what about FocusBundleActivator, FocusManager and ConfigurationService?